src/widgets/heightandweight/heightandweight.resource.ts

```diff
@@ -12,11 +12,13 @@ export function getDimensions(patientId: string) {
 }
 
 function getDimensionsObservations(patientId: string) {
-  return openmrsObservableFetch(
+  return openmrsObservableFetch<DimensionFetchResponse>(
     `${fhirConfig.baseUrl}/Observation?subject:Patient=${patientId}&code=${WEIGHT_CONCEPT},${HEIGHT_CONCEPT}`
   ).pipe(
-    map(({ data }) => data["entry"]),
-    map((entries: []) => entries.map((entry: any) => entry.resource)),
+    map(({ data }) => data.entry),
+    map((entries: Array<Dimension>) =>
```

Comment:
I think there should not be an explicit `Array<Dimension>` specified here, it should be correctly inferred by TS.